### ISSUE TITLE
JS: Discuss the History of JavaScript

### ISSUE THREAD

**Issue by paramsiddharth** [girlscript/winter-of-contributing#70]
# Discuss the History of JavaScript
Talk about the historical development of JavaScript

## Directory
```
Topics/2.History
```

## Description
Refer to the history of the usage of scripting in front-end
and the [Wikipedia](https://en.wikipedia.org/wiki/JavaScript) page
for JavaScript.

## Notes
It is always considered a good practice to cite your sources.
Make sure you do that in your PR.

**Comment by anmol20112000** [comment]
Heyy!! I want to work on this... Plzz assign this to me...

**Comment by harmankaler2000** [comment]
Hi! I'd like to contribute to this issue and talk about the map of JavaScript and it's history, How it came into being and why it's used.
Thank you
Harmandeep Kaur Kaler
FEWD - Batch 2

**Comment by paramsiddharth** [comment]
> Hi! I'd like to contribute to this issue and talk about the map of JavaScript and it's history, How it came into being and why it's used.
> Thank you
> Harmandeep Kaur Kaler
> FEWD - Batch 2

@harmankaler2000 Glad to see you willing to contribute! Could you please confirm from the administration if
FEWD contributors are eligible to contribute to JavaScript too? If yes, I'll be happy to consider your PR. Meanwhile
I'll assign you the issue.

**Comment by harmankaler2000** [comment]
Yes sir, I'll ask the administrators if I am eligible to take part in JavaScript under FEWD.

**Comment by harmankaler2000** [comment]
@paramsiddharth Sir, I talked to my supervisor, he clarified that it is a different domain than FEWD. so I am not eligible under GWOC to contribute to it.

**Comment by sushovanb02** [comment]
@paramsiddharth sir, I want to work on this issue. It would be really nice to present the history of JavaScript. Looking forward for an exciting journey...
Sushovan Banerjee
JavaScript batch-3

**Comment by Korak-Sengupta** [comment]
Wish to work on it 🙂

**Comment by AbhinandanAdhikari** [comment]
Sir, I want to work on it

**Comment by sakkshm26** [comment]
Hey, can I work on this issue?

**Comment by Akshita-Sharma-25** [comment]
Myself Akshita and i would like to contribute towards this issue.

**Comment by iam-sakshi** [comment]
Hello! sir I wish to work on this issue.

**Comment by paramsiddharth** [comment]
@iam-sakshi Please note that a document has already been contributed to this topic so you may either choose to extend it with more data or make a contribution in video/audio format (if not already done).
